keyboards/handwired/wulkan/README.md

```diff
@@ -0,0 +1,14 @@
+# wulkan
+
+Handwired 40% keyboard build with Proton C.
+
+Keyboard Maintainer: [Napoleon Wulkan](https://github.com/wulkan)  
+Hardware Supported: Proton C  
+Hardware Availability: [OLKB.com](https://olkb.com)
+
+
+Make example for this keyboard (after setting up your build environment):
+
+    CFLAGS="-Wno-error=deprecated" make handwired/wulkan:default:dfu-util
```

Comment:
Is there a reason for the CFLAG here?

Reply:
Yes, I did not manage to build against Proton C without it, running with gcc version 9.1.0. Seen other people having the same problem and this solves it. And it don't break anything for version 8 so I kept it in the readme file, but I could put an explanation for it in there as well if you like

Reply:
Well, QMK currently doesn't support 9.1.0 for that reason. It might be better to remove it and downgrade to 8.3.0.

Reply:
Will remove it then, you know if there is any plans for supporting 9.1.0 in the near future? Not a big issue for me, just curious :)